erigon-lib/downloader/downloader.go

```diff
@@ -2718,6 +2718,9 @@ func openClient(ctx context.Context, dbDir, snapDir string, cfg *torrent.ClientC
 	//})
 	cfg.DefaultStorage = m
 
+	dnsResolver := &downloadercfg.DnsCacheResolver{RefreshTimeout: 24 * time.Hour}
+	cfg.TrackerDialContext = dnsResolver.DialContext
```

Comment:
`dnsResolver` object just created without setting DialContext field. Did you mean opposite assignment? Or did you forgot to set `DialContext` field?

Reply:
DialContext is a method 🧙